test/build_samples.go

```diff
@@ -214,6 +214,105 @@ spec:
     image: image-registry.openshift-image-registry.svc:5000/example/buildpacks-app
 `
 
+// BuildCBSWithFakeURLWithAnnotation defines a Build with a
+// ClusterBuildStrategy and a fake url
+const BuildCBSWithFakeURLWithAnnotation = `
```

Comment:
It concerns me seeing all of this yaml crammed into a go variable. We have no way to validate it. I would much rather see us use yaml files and maybe patch the yaml when needed? Just a thought.

Reply:
this is an approach to reduce the amount of YAMLs, which I tend to try to avoid. But you have a point, the amount of constants are increasing. We should do a clean-up PR at some point later where we revisit this.